BiweeklyMeetings/2022-06-07.md

```diff
@@ -0,0 +1,370 @@
+
+**M. Àngels** -:  So, do you want to put it on hold for a few meetings and maybe after the hard fork before trying to move it again?
+
+**Matthias** -: Yeah. I mean, we'll need to poke basically IOG constantly, I guess, until we get a reaction. There was also this new simulation engine that was supposed to come out. So maybe that would be a good time to look again into the CIP once this simulation engine is out, although there is no clear timeline, so this could also very much be six months or a year from now, which I don't know.
+
+**Matthias** -: I see also there is a comment from last week that I missed from an SPO. Which is quite lengthy. So from the quick reads, it seems to be again, debating on whether or not the CIP makes things better regarding decentralization. Because there is also this question at this stage, and this is what was brought up by the researchers when they looked at CIPs, was that from a decentralization standpoint and again, theory standpoint, this was an approach that was discussed during the early days of Shelley's and that was rolled out because not good enough at maintaining decentralization. So multiples were also considered in designing the first place. Yeah.
+
+**Matthias** -: I mean, the situation we are in today was kind of already included. So there is probably better to do but whether CIP 50 is going in that direction, it's also good question and it's not clear right now. We don't have any quite clear proof that it is actually going in the right direction. So far it seems to be more of in sentiment than an actual well calculations or simulation of that new theory. Maybe that's something which would be worth trying on the site network. But importantly, we don't really have that.
+
+**Matthias** -: And the stake on the test net would be for instance, much more different because it's all free money and people don't have the same behavior, therefore. So, yeah, I don't know. We can try to bring IOG back again, see what's the answer and keep to have short update on that every week or every two weeks.
+
+**M. Àngels** -:  Okay. So we are going to keep it through for next meeting agenda?
+
+**Matthias** -: I would say for now, yes. And let's see, we probably won't get any update from put outputs before the Hard Fork. Because I guess all minds are focused on that, but we might a bit later.
```

Comment:
```suggestion
**Matthias** -: I would say for now, yes. And let's see, we probably won't get any update from Input Output before the Hard Fork. Because I guess all minds are focused on that, but we might a bit later.
```